init.lua

```diff
@@ -45,6 +45,12 @@ local function normalize(img, opts)
   return img
 end
 
+-- Set the URL of the listening server
+function M.server(config)
```

Comment:
I agree that `server` is not a very apt name. I was actually thinking a lot about it and I couldn't find a better one. I'm new to Lua, so I don't know what's the convention here. If it was Java, I would have named it as a setter method. 

I guess `display.use` might work, although it's somewhat ambiguous (the display can 'use' many things), and it doesn't hint to the server configuration.